controllers/status.go

```diff
@@ -240,6 +243,58 @@ func (r *PerformanceProfileReconciler) getKubeletConditionsByProfile(profile *pe
 	return r.getDegradedConditions(conditionKubeletFailed, latestCondition.Message), nil
 }
 
+func (r *PerformanceProfileReconciler) getTunedConditionsByProfile(profile *performancev2.PerformanceProfile) ([]conditionsv1.Condition, error) {
+	tunedProfileList := &tunedv1.ProfileList{}
+	if err := r.List(context.TODO(), tunedProfileList); err != nil {
```

Comment:
Where are we doing the match between the performance profile tuned name and the existing tuneds in the cluster here?

Reply:
We don't. We should look for a match between the node's name and tuned's profile name, as you can see [here](https://github.com/openshift-kni/performance-addon-operators/pull/583/files#diff-e8b328ffb5ced885894c80b942846601597e05020f831c2b1442d8fb8a654c8dR325)
The node itself associate with the PP via the ```nodeSelector```

Reply:
right. you are getting all the profiles.